src/Models/Like.cs

```diff
@@ -12,6 +12,6 @@ public class Like
         [Key]
         public Guid Id { get; set; }
         public Guid UserId { get; set; }
-        public Solution Solution { get; set; }
+        public virtual Solution Solution { get; set; }
```

Comment:
Why is this virtual?

Reply:
https://stackoverflow.com/questions/8542864/why-use-virtual-for-class-properties-in-entity-framework-model-definitions